scripts/mailing_lists/migrate_project_mailing_lists.py

```diff
@@ -0,0 +1,115 @@
+# -*- coding: utf-8 -*-
```

Comment:
Added in https://github.com/CenterForOpenScience/osf.io/pull/5645/commits/dc99ed6994df891f29345cdd4d9b2289b5b2976c